### DISCUSSION TITLE
Billing information for AWS Codebuild (ideally the whole Codesuite)

### DISCUSSION THREAD

**Discussion by jolo-dev** [tailwarden/komiser#1154] [Ideas]
Hi there,
In our company, we use AWS Codebuild quite much. It would be nice to get their pricing information: https://aws.amazon.com/codebuild/pricing/

What do you think?

**Comment by mlabouardy** [comment]
Hey @jolo-dev
That would be great 
cc @jakepage91 @Azanul @Traxmaxx

**Reply by jakepage91** [reply]
Will put this to the mantianers and contributors and we can see if we can get this prioritised!
